cmd/mtail/main.go

```diff
@@ -147,6 +148,10 @@ func main() {
 		glog.Infof("no poll interval specified; defaulting to 250ms poll")
 		*pollInterval = time.Millisecond * 250
 	}
+	if *pollLogInterval == 0 {
+		glog.Infof("no poll interval specified; defaulting to pollInterval")
```

Comment:
Once this is changed to be the pattern interval, then this and line 148 should explain which poll interval is being warned about.

Reply:
I have updated the description, please take a look.